src/RichText/ErrorBoundary.tsx

```diff
@@ -19,12 +19,12 @@ export class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoun
 		};
 	}
 
-	static getDerivedStateFromError(error: any): Partial<ErrorBoundaryState> | null {
+	static getDerivedStateFromError(): Partial<ErrorBoundaryState> | null {
 		// Update state so the next render will show the fallback UI.
 		return { hasError: true };
 	}
 
-	componentDidCatch(error: Error, info: React.ErrorInfo): void {
```

Comment:
What was the problem here? That info was unused?

Removing it is fine, another approach would be renaming it with underscore prefix.

_info: React.ErrorInfo